packages/docs/src/routes/docs/(qwik)/components/resource/index.mdx

```diff
@@ -22,6 +22,38 @@ This method allows you to generate computed values asynchronously. The asynchron
 
 Just like all the `use-` methods, it must be called within the context of [`component$()`](/docs/(qwik)/components/overview/index.mdx#component), and all the [hook rules apply](/docs/(qwik)/components/lifecycle/index.mdx#use-method-rules).
 
+Here is a very simple example of using `useResource$()` to fetch a random joke:
+
+```tsx
+import { component$, useResource$, Resource } from '@builder.io/qwik';
+
+export default component$(() => {
+  const jokes = useResource$<{ value: string }[]>(
+    async ({ track, cleanup }) => {
+      const resp = await fetch('https://api.chucknorris.io/jokes/random')
+      return (await resp.json()) as { value: string };
+    }
+  );
+
+  return (
+    <Resource
+      value={jokes}
+      onPending={() => <>loading...</>}
```

Comment:
here we could use a `<span>` or `<div>`, etc instead. guess not all devs are aware of fragments.

Reply:
works for me ✅